students/byoungju/users/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.urls import path
+from users.views import UserView
+
+urlpatterns = [
+    path("user", UserView.as_view()),
```

Comment:
이렇게 되면 최종적인 엔드포인트는 users/user의 형태가 되겠네요. 불필요하게 중복되니 빈스트링을 넣어 users만으로 연결될 수 있게 수정해주세요!
```suggestion
    path("", UserView.as_view()),
```

Reply:
네! 빈 형태로 수정했습니다